### DISCUSSION TITLE
How to change highlighting for folded headings?

### DISCUSSION THREAD

**Discussion by s-tlm** [NvChad/NvChad#1873] [Q&A]
I'm currently using the Aquarium theme which highlights folded headings as light grey.

<img width="487" alt="image" src="https://user-images.githubusercontent.com/72781965/228966835-c5c15741-110e-4ab5-b5dd-cb753a168319.png">

I'm looking to change the highlight colour so its easier to see. Which variable should I edit in my custom `highlights.lua`?

**Comment by siduck** [comment]
@s-tlm please read https://nvchad.com/docs/config/theming
